src/i18n/es/index.js

```diff
@@ -0,0 +1,156 @@
+import print from './print';
+import tester from './tester';
+export default {
+  es: {
+    message: {
+      ...potato,
+      help: {
+        label: 'Abrir tutorial'
+      },
+      print: { ...print },
+      tester: { ...tester },
+      keyboard: {
+        label: 'teclado'
+      },
+      layout: {
+        label: 'layout'
+      },
+      keymapName: {
+        label: 'Nombre de keymap',
+        placeholder: 'nombre de keymap personalizado'
+      },
+      loadDefault: {
+        label: 'Cargar Original',
+        title: 'Cargar valor por defecto para layout de Firmware QMK'
+      },
+      compile: {
+        label: 'Compilar',
+        title: 'Compilar el keymap'
+      },
+      downloadKeymap: {
+        title: 'Descargar solamente keymap.c',
+        label: 'solo keymap'
+      },
+      downloadSource: {
+        title: 'Descargar código de Firmware QMK',
+        label: 'Código completo'
+      },
+      downloadJSON: {
+        title: 'Exportar archivo JSON de Keymap QMK',
+        label: 'Keymap.JSON'
+      },
+      importJSON: {
+        title: 'Importar archivo JSON de Keymap QMK'
+      },
+      importUrlJSON: {
+        title: 'Importar archivo JSON de Keymap QMK desde URL'
+      },
+      printKeymap: {
+        title: 'Imprimir capas de keymap',
+        label: 'Imprimir keymap'
+      },
+      testKeys: {
+        title: 'Probar entrada del teclado',
+        label: 'Probar teclado'
+      },
+      downloadFirmware: {
+        label: 'Firmware',
+        title: 'Descargar archivo de firmware para instalar'
+      },
+      flashFirmware: {
+        label: 'Auto-Instalar',
+        title: 'Automáticamente instalar Firmware compilado al MCU'
+      },
+      flashFile: {
+        label: 'Instalar personalizado',
+        title: 'Instalar archivo elegido al MCU'
+      },
+      ColorwayTip: {
+        title: 'Ctrl + Alt + N para cambiar al próximo color'
+      },
+      layer: {
+        label: 'Capa',
+        confirm: '¿Estás seguro de que quieres borrar la capa?',
+        title: 'Borrar capa'
+      },
+      keymap: {
+        label: 'Keymap'
+      },
+      downloadToolbox: {
+        label: 'Descargar QMK Toolbox'
+      },
+      keycodes: {
+        label: 'Códigos de teclas'
+      },
+      keycodesRef: {
+        label: 'Referencia de códigos de tecla'
+      },
+      keycodesTab: {
+        ANSI: {
+          label: 'ANSI'
+        },
+        'ISO/JIS': {
+          label: 'ISO/JIS'
+        },
+        Quantum: {
+          label: 'Quantum'
+        },
+        KeyboardSettings: {
+          label: 'Configuración de teclado'
+        },
+        AppMediaMouse: {
+          label: 'App, Media y Mouse'
+        }
+      },
+      settingsPanel: {
+        title: 'Configuración de Configurator',
+        fastInput: {
+          label: 'Entrada rápida',
+          title: 'ctrl + alt + f',
+          help:
+            'Ingresar teclas por media del teclado sin hacer clic en cada posición.'
```

Comment:
```suggestion
            'Ingresar teclas por medio del teclado sin hacer click en cada posición.'
```
Ops! I didn't see this typo before

Reply:
I just pushed up a commit to address these. I had actually frequently seen click spelled "clic"! But I guess that's pretty colloquial, so it'll be better to take the full anglicism.

Reply:
You are completely right, I searched for the word and officially clic is the one accepted despite you see click everywhere specially among young people (that's why I thought it was a typo).